docassemble/MassAppealsCourt/data/questions/appeals-basic-questions.yml

```diff
@@ -22,7 +22,7 @@ id: appeals basic questions intro screen
 question: |
   ${interview_short_title}: Mass Access Project
 subquestion: |
-  % if form_approved_for_email_filing:
+  % if form_approved_for_email_filing or (form_uses_efiling and efile_setup):
```

Comment:
Can we provide a placeholder value for `efile_setup`, as a `DAEmpty`? Would mean we require files to be included in a specific order, but that's pretty manageable.

Reply:
Not a huge fan of that, because we have an `efiling.yml` file in there that we need to include after other EFSPIntegration files (we overwrite asking the user if they want to e-file, and the logic of how to get the `court_id`, which defaults to `trial_court`). It might work, but I think the order would have to be:

* `MassAppealsCourt:appeals-basic-questions.yml`
* `EFSPIntegration:efiling_integration.yml`
* `MassAppealsCourt:efiling.yml`

Something that no one will know without knowing all of the contents of each of those files, and is fairly precarious IMO.

I'd rather move this to `efiling.yml`, I can do that instead.

Reply:
Alternatively, I can just remove `efile_setup` here. It indicates that the DA server can communicate correctly with the EfileProxy. However, once an interview is written to use e-filing, it'll likely be advertised as such elsewhere, so it'd be confusing for users if they came to the interview expecting e-filing and finding no indication of it.

We already do include a screen that says "You will *not* be able to e-file" if something goes wrong at runtime, and we'll always want that. I don't think we need to both change the introduction screen and show the "you will *not*" screen (which only comes 2 screens after, no questions between intro and there).